### PR TITLE
Added recaptcha support

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: Controller/FormController.php [hunk 1]
```diff
@@ -194,7 +193,11 @@ public function cgetTemplateAction(Request $request)
         $types = Dynamic::$TYPES;
 
         if (!empty($mailChimpLists)) {
-            $types = array_merge(Dynamic::$TYPES, [Dynamic::TYPE_MAILCHIMP]);
+            $types[] = Dynamic::TYPE_MAILCHIMP;
+        }
+
+        if (class_exists('EWZ\Bundle\RecaptchaBundle\Form\Type\RecaptchaType')) {
+            $types[] = Dynamic::TYPE_RECAPTCHA;
```
Comment (wachterjohannes) [inline Controller/FormController.php:200]: your are currently not able to include form-types from outside?

File: Form/Type/DynamicFormType.php [hunk 2]
```diff
@@ -154,6 +154,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 case Dynamic::TYPE_MAILCHIMP:
                     $type = CheckboxType::class;
                     break;
+                case Dynamic::TYPE_RECAPTCHA:
+                    $type = 'EWZ\Bundle\RecaptchaBundle\Form\Type\RecaptchaType';
+                    $recaptchaConstraintClass = 'EWZ\Bundle\RecaptchaBundle\Validator\Constraints\IsTrue';
```
Comment (wachterjohannes) [inline Form/Type/DynamicFormType.php]: why not using directly?

File: Resources/translations/messages.fr.xliff [hunk 3]
```diff
@@ -14,112 +14,6 @@
...
-            <trans-unit id="31">
-                <source>l91_sulu_form.type.checkboxMultiple</source>
-                <target>Checkboxes</target>
-            </trans-unit>
-            <trans-unit id="32">
-                <source>l91_sulu_form.type.dropdown</source>
-                <target>Select</target>
-            </trans-unit>
-            <trans-unit id="33">
-                <source>l91_sulu_form.type.dropdownMultiple</source>
```
Comment (wachterjohannes) [inline Resources/translations/messages.fr.xliff:108]: ??? why removing all of them?

File: Controller/FormController.php (hunk 1 above)
Comment (alexander-schranz) [reply Controller/FormController.php:200]: yes currently not. #78

File: Resources/translations/messages.fr.xliff (hunk 3 above)
Comment (alexander-schranz) [reply Resources/translations/messages.fr.xliff:108]: as i see it was added for the FormControlller `trans` function did include there to use the `backend` domain so this translates dont need to be duplicated

File: Form/Type/DynamicFormType.php (hunk 2 above)
Comment (alexander-schranz) [reply Form/Type/DynamicFormType.php]: afraid what happen when the class dont exist and included over `use`?

Comment (wachterjohannes) [reply Form/Type/DynamicFormType.php]: write a hint here and in the issue.

File: Resources/translations/messages.fr.xliff (hunk 3 above)
Comment (wachterjohannes) [reply Resources/translations/messages.fr.xliff:108]: +1

**Pull request merged**